lineman/app/components/navbar/navbar_user_options.coffee

```diff
@@ -3,5 +3,9 @@ angular.module('loomioApp').directive 'navbarUserOptions', ->
   restrict: 'E'
   templateUrl: 'generated/components/navbar/navbar_user_options.html'
   replace: true
-  controller: ($scope, CurrentUser) ->
+  controller: ($scope, CurrentUser, Records, $http) ->
```

Comment:
Also, we in other places inject the $window object instead of using window directly, for testability reasons. 

Injecting the window object means that in our unit tests we can say 'inject a $window instance which has a Loomio.currentUserId variable on it', and not pollute the global namespace for the other tests. (it may not apply here, but I'd rather not give a bad example for future developers)